.forge-snapshots/swap skips hook call is hook is caller.snap

```diff
@@ -0,0 +1 @@
+156294
```

Comment:
You tested this gas against the old impl?

Reply:
oops i didnt reply
on main: 157001